templates/initd.j2

```diff
@@ -1,4 +1,5 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Bash can't be here. Busybox > ash etc.. it's better to have compatible #!/bin/sh